index.d.ts

```diff
@@ -487,12 +509,12 @@ declare module "@2colors/esphome-native-api" {
             address: number,
             handle: number,
             value: Uint8Array,
-            response = false
+            response?: boolean
         ): Promise<BluetoothGATTWriteResponse>;
         notifyBluetoothGATTCharacteristicService(
             address: number,
             handle: number
-        ): Promise<BluetoothGATTNotifyResponse>;
+        ): Promise<BluetoothGATTNotifyDataResponse>;
```

Comment:
This is not correct. When subscribing to notifications ESPHome will emit a BluetoothGATTNotifyResponse to confirm it is done, BluetoothGATTNotifyDataResponse is only emitted when ESPHome notifies with the value.

Reply:
sorry missed that because type for BluetoothGATTNotifyResponse was missing. Fixed now